### PR TITLE
chore: remove stage-repo in pom due to release done & update mail rule

### PR DESCRIPTION
**Pull request by z7658329**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -48,9 +48,9 @@ github:
         required_approving_review_count: 2
 
 notifications:
-  pullrequests_status: dev@hugegraph.apache.org
+  #pullrequests_status: dev@hugegraph.apache.org
   # before use the config, we should ensure the "mail" address has set well (exist)
   #pullrequests_comment: issues@hugegraph.apache.org
   #issues: issues@hugegraph.apache.org
-  #discussions: issues@hugegraph.apache.org
+  discussions: issues@hugegraph.apache.org
```
Comment (imbajin) [inline .asf.yaml:55]: use`dev` mail, we don't have issue mail now.. check the comment

**Review by imbajin — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -668,19 +668,4 @@
             </build>
         </profile>
     </profiles>
-
-    <repositories>
```
Comment (javeme) [inline pom.xml:672]: can we add commit messages for why to remove it?

File: .asf.yaml [hunk 3]
```diff
@@ -48,9 +48,9 @@ github:
         required_approving_review_count: 2
 
 notifications:
-  pullrequests_status: dev@hugegraph.apache.org
+  #pullrequests_status: dev@hugegraph.apache.org
   # before use the config, we should ensure the "mail" address has set well (exist)
   #pullrequests_comment: issues@hugegraph.apache.org
   #issues: issues@hugegraph.apache.org
-  #discussions: issues@hugegraph.apache.org
+  discussions: dev@hugegraph.apache.org
```
Comment (javeme) [inline .asf.yaml:55]: can add a comment to explain discussions?

Comment (imbajin) [reply .asf.yaml:55]: > can add a comment to explain discussions?

maybe add a link is enough?
image

File: pom.xml (hunk 2 above)
Comment (imbajin) [reply pom.xml:672]: > can we add commit messages for why to remove it?

Update: modify it in title

File: .asf.yaml (hunk 3 above)
Comment (javeme) [reply .asf.yaml:55]: get it

**Review by javeme — DISMISSED** [review]

**Comment by imbajin** [comment]
@z7658329 
Update: Found the `dev` mail still contains a lot of `issues` notification (which is not in expectations)
1. maybe we need config the `issue: xxx`, otherwise ASF will use a default config
2. apply a new email for `issues@hugegraph.apache.org` or redirect it to `commits@hugegraph.apache.org` first

refer [.asf.yaml](https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features) to get the **accurate answer**, then we sync the config in all other repos

**Review by imbajin — DISMISSED** [review]

**Review by imbajin — APPROVED** [review]

File: .asf.yaml
Comment (imbajin) [inline .asf.yaml:1]: Already apply for the `issues` mailbox, record the info

**Review by javeme — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**